kview/src/main/java/dev/ikm/komet/kview/mvvm/viewmodel/StampViewModel2.java

```diff
@@ -36,23 +66,95 @@ public enum StampProperties {
 
     public StampViewModel2() {
         super();
-        addProperty(ENTITY, (EntityFacade) null);
-        addProperty(PREV_STAMP, (Stamp) null);
+        addProperty(CURRENT_STAMP, (Stamp) null);
         addProperty(STATUS, State.ACTIVE);
         addProperty(MODULE, (ConceptEntity) null);
         addProperty(PATH, (ConceptEntity) null);
 
-        // TODO:
-//        addValidator(SAME_AS_PREVIOUS, )
+        addProperty(IS_STAMP_VALUES_THE_SAME, true);
```

Comment:
I believe this is optional. If you are using it to be set. in the validator code it should change the value to true of false. in line after 76. `vm.setPropertyValue(IS_STAMP_VALUES_THE_SAME, same);`

Reply:
Nevermind I see this set in the custom method to check for 'if same'.